src/index.html

```diff
@@ -9,6 +9,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home"><img src="./images/logo.png" alt="logo"></a>
+      </div>
+      <nav class="menu">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#home">apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="hover"
+              href="#">
+              laptops & computers
+            </a>
```

Comment:
Check good code examples in checklist.
```suggestion
            <a
              class="nav__link"
              data-qa="hover"
              href="#"
            >
              laptops & computers
            </a>
```